pkg/api/v1beta1/conversion.go

```diff
@@ -707,6 +707,38 @@ func init() {
 			}
 			return nil
 		},
+
+		func(in *newer.PullPolicy, out *PullPolicy, s conversion.Scope) error {
+			switch *in {
+			case newer.PullAlways:
+				*out = PullAlways
+			case newer.PullNever:
+				*out = PullNever
+			case newer.PullIfNotPresent:
+				*out = PullIfNotPresent
+			case "":
+				*out = ""
+			default:
+				//FIXME: Should we be doing validation here?  How does this play into the UX?
```

Comment:
Technically we shouldn't, unless the only place the check can be done is here.  I think we might eventually want a revalidation on the api version, then validate on internal.